pkg/webui/console/views/app/index.js

```diff
@@ -30,26 +31,38 @@ import FullViewError from '../error'
 
 import style from './app.styl'
 
+const m = defineMessages({
+  title: '%s - {siteTitle} - {siteName}',
+})
+
 @withEnv
 @bind
 export default class ConsoleApp extends React.Component {
   render () {
     const {
       env,
     } = this.props
+    
+    const pageEnv = {
+      site_name: window.SITE_NAME,
+      site_title: window.SITE_TITLE,
+    }
 
     const pageData = pageDataSelector(env)
 
     if (pageData && pageData.error) {
       return <FullViewError error={pageData.error} />
     }
-
     return (
       <ErrorView ErrorComponent={FullViewError}>
         <div className={style.app}>
-          <Helmet
-            titleTemplate="%s - Console - TTN Stack"
-            defaultTitle="TTN Stack Console"
+          <IntlHelmet
+            titleTemplate={m.title}
```

Comment:
```suggestion
            titleTemplate={`%s - ${siteTitle} - ${siteName}`}
```
Just use a template string here.